------------------------------------------------------------
revno: 13944
revision-id: squid3@treenet.co.nz-20151103144438-vl05r5uf1vxh6m3s
parent: squid3@treenet.co.nz-20151101104351-zxake0uhme9d0z53
committer: Amos Jeffries <squid3@treenet.co.nz>
branch nick: 3.5
timestamp: Tue 2015-11-03 06:44:38 -0800
message:
  Make FATAL messages have a consistent prefix
------------------------------------------------------------
# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: squid3@treenet.co.nz-20151103144438-vl05r5uf1vxh6m3s
# target_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# testament_sha1: fda115516a1d92aef4663ea12a6dde79b06a9293
# timestamp: 2015-11-03 14:48:27 +0000
# source_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# base_revision_id: squid3@treenet.co.nz-20151101104351-\
#   zxake0uhme9d0z53
# 
# Begin patch
=== modified file 'src/cache_cf.cc'
--- src/cache_cf.cc	2015-02-18 08:50:00 +0000
+++ src/cache_cf.cc	2015-11-03 14:44:38 +0000
@@ -4022,7 +4022,7 @@
     }
 
     if (stat(path, &sb) < 0) {
-        debugs(0, DBG_CRITICAL, (opt_parse_cfg_only?"FATAL ":"") << "ERROR: " << name << " " << path << ": " << xstrerror());
+        debugs(0, DBG_CRITICAL, (opt_parse_cfg_only?"FATAL: ":"ERROR: ") << name << " " << path << ": " << xstrerror());
         // keep going to find more issues if we are only checking the config file with "-k parse"
         if (opt_parse_cfg_only)
             return;

